pgrx/src/datum/range.rs

```diff
@@ -164,16 +164,16 @@ where
         match self.as_ref() {
             None => write!(f, "empty"),
             Some((RangeBound::Infinite, RangeBound::Infinite)) => write!(f, "(,)"),
-            Some((RangeBound::Infinite, RangeBound::Inclusive(v))) => write!(f, "(,{}]", v),
-            Some((RangeBound::Infinite, RangeBound::Exclusive(v))) => write!(f, "(,{})", v),
+            Some((RangeBound::Infinite, RangeBound::Inclusive(v))) => write!(f, "(,{v}]"),
+            Some((RangeBound::Infinite, RangeBound::Exclusive(v))) => write!(f, "(,{v})"),
 
-            Some((RangeBound::Inclusive(v), RangeBound::Infinite)) => write!(f, "[{},)", v),
-            Some((RangeBound::Inclusive(l), RangeBound::Inclusive(u))) => write!(f, "[{},{}]", l, u),
-            Some((RangeBound::Inclusive(l), RangeBound::Exclusive(u))) => write!(f, "[{},{})", l, u),
+            Some((RangeBound::Inclusive(v), RangeBound::Infinite)) => write!(f, "[{v},)"),
+            Some((RangeBound::Inclusive(l), RangeBound::Inclusive(u))) => write!(f, "[{l},{u}]"),
+            Some((RangeBound::Inclusive(l), RangeBound::Exclusive(u))) => write!(f, "[{l},{u})"),
```

Comment:
This reads somewhat confusingly when it is inlined, I would prefer
```suggestion
            Some((RangeBound::Inclusive(lo), RangeBound::Inclusive(hi))) => write!(f, "[{lo},{hi}]"),
            Some((RangeBound::Inclusive(lo), RangeBound::Exclusive(hi))) => write!(f, "[{lo},{hi})"),
```